docs/useListContext.md

```diff
@@ -142,8 +142,7 @@ const CustomFilter = () => {
 
     const handleSubmit = (event) => {
         event.preventDefault();
-        // The 3rd parameter disables the debounce ⤵
-        setFilters(filterFormValues, undefined, false);
+        setFilters(filterFormValues, undefined);
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏same

```suggestion
        setFilters(filterFormValues);
```